sdk/storage/azblob/service/client.go

```diff
@@ -123,6 +123,14 @@ func (s *Client) DeleteContainer(ctx context.Context, containerName string, opti
 	return containerDeleteResp, err
 }
 
+// RestoreContainer restores soft-deleted container
+// Operation will only be successful if used within the specified number of days set in the delete retention policy
+func (s *Client) RestoreContainer(ctx context.Context, deletedContainerName string, options *RestoreContainerOptions) (RestoreContainerResponse, error) {
```

Comment:
Here as well. Required parameters should be in the signature.

Reply:
Added